src/types/transaction.js

```diff
@@ -53,13 +54,25 @@ export type TransactionCommon = {|
   recipient: string,
   useAllAmount?: boolean,
   subAccountId?: ?string,
+  feesStrategy?: "low" | "medium" | "high" | "advanced",
 |};
 
 export type TransactionCommonRaw = {|
   amount: string,
   recipient: string,
   useAllAmount?: boolean,
   subAccountId?: ?string,
+  feesStrategy?: "low" | "medium" | "high" | "advanced",
```

Comment:
i think we should make this a non optional.

Reply:
At the moment most of coins doesn't have the need to set this option